pkg/cmd/workspace/ssh.go

```diff
@@ -13,27 +13,25 @@ import (
 	"github.com/daytonaio/daytona/pkg/apiclient"
 	workspace_util "github.com/daytonaio/daytona/pkg/cmd/workspace/util"
 	"github.com/daytonaio/daytona/pkg/ide"
-	"github.com/daytonaio/daytona/pkg/views"
 	"github.com/daytonaio/daytona/pkg/views/workspace/selection"
 
-	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 )
 
 var SshCmd = &cobra.Command{
-	Use:     "ssh [WORKSPACE] [PROJECT]",
+	Use:     "ssh [WORKSPACE] [PROJECT] [CMD...]",
 	Short:   "SSH into a project using the terminal",
-	Args:    cobra.RangeArgs(0, 2),
+	Args:    cobra.ArbitraryArgs,
 	GroupID: util.WORKSPACE_GROUP,
-	Run: func(cmd *cobra.Command, args []string) {
+	RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Is "RunE" leftover now that we don't call the command directly from other places?

Reply:
Removed